airbyte-integrations/connectors/source-paypal-transaction/metadata.yaml

```diff
@@ -20,7 +20,7 @@ data:
   name: Paypal Transaction
```

Comment:
This file should be reverted

Reply:
yeah, that's a bug in our approve-and-merge pipeline. The automatic formatting should have kicked in but it didn't, and as a result, another random change updated that file: https://github.com/airbytehq/airbyte/pull/31610/files